http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1138,11 +1138,16 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         }
         .getOrElse(gkeAppConfig.chart)
       release <- lastUsedApp.fold(
-        KubernetesName.withValidation(
-          s"${uid}-${gkeAppConfig.releaseNameSuffix.value}",
-          Release.apply
-        )
+        KubernetesName
+          .withValidation(
+            s"${uid}-${gkeAppConfig.releaseNameSuffix.value}",
+            Release.apply
+          )
       )(app => app.release.asRight[Throwable])
+      services =
+        if (cloudContext.cloudProvider == CloudProvider.Azure) {
+          gkeAppConfig.kubernetesServices.appended(RelayListenerChartConfig.service)
```

Comment:
I thought about making the Relay Listener a full-fledged `KubernetesAppConfig`. But it doesn't share all attributes with the other apps -- it's more a thing implicitly attached to Azure apps, than a standalone app config. So here, if the app is Azure, I'm appending the `relay-listener` service to persist to the DB.

Reply:
That makes sense, at some point, should we also think about moving the rest of the setup chart into its own config outside of tera-app and deploy it in a similar manner?

Reply:
That may make sense, the terra-helmfile repo is where most of our "internal" charts are landing these days.

One note: after this change, we no longer need to deploy the setup chart for Azure apps (the listener was the only thing we were using). So it's more of a GCP-only thing now.